frontend/js/components/DataTable/DataTable.component.js

```diff
@@ -184,9 +184,9 @@ class DataTable extends HTMLElement {
         let maxPossiblePageNumber =  Math.ceil(tableColumnsArrayLength / columnPerPage);
         let tableColumnsArrayCurrent = [];
         let z = 0;
-        for(let i= pageNumber*columnPerPage ;i < Math.min(tableColumnsArrayLength, pageNumber*columnPerPage+15) ; i++ )
-        {
-            tableColumnsArrayCurrent[z]=tableColumnsArray[i];
+        for (let i = pageNumber * columnPerPage; i < Math.min(tableColumnsArrayLength, pageNumber * columnPerPage + 15); i++) {
+            if (tableColumnsArray[i] != 'synth_id')
```

Comment:
why is this ignored ? Do we not render the synth id ?

Reply:
As per the current structure, we are fetching the source/spanner column information using the source-spanner map. Here there is no column in the source and hence it can cause issues. Moreover, there is no source column to display in the grid for the synthetic key.
Do we need to display the synthetic key even though we do not have any column at the source?